runtime/early/extra-mappings.vim

```diff
@@ -0,0 +1,9 @@
+" Extra default keymaps and keyboard behavior
+" see :help nvim-extra-mappings
+
+tnoremap <esc><esc> <C-\><c-n>
+nnoremap Y y$
```

Comment:
I guess you're right, I just compared `:h v_y` and `:h y` and noticed the lack of a `{motion}` argument on the former. My mistake.